### ISSUE TITLE
Accessibility disable option

### ISSUE THREAD

**Issue by oguzveozturk** [DominikButz/SwipeCellSUI#5]
When accessibility open VoiceOver says the systemImage name every time on the cell.  I can avoid this using anyView instead of systemImage. But I think we shouldn't use[ anyView inside ForEach loop](https://developer.apple.com/videos/play/wwdc2021/10022/)s.

**Comment by DominikButz** [comment]
Hi, I'm sorry but I'm not familiar with Voice over. I have just tried it out and it seems that swiping does not work at all with accessibility voice over, at least I don't know how. I also tried it out in Apple Mail (where you can swipe each email list cell - left and right), it also does not work. Please provide more information on how I can help exactly. You can also fork my project and then make a pull request after modifying the code.
